### PR TITLE
update stable version and its hunterization

### PR DESCRIPTION
**Pull request by tonyrewin**

### CODE REVIEW

File: bin/alex_model.inl [hunk 1]
```diff
@@ -1,289 +0,0 @@
-// Alex's Model
```
Comment (ruslo) [inline bin/alex_model.inl]: I do expect only CMake part update. If you want to update the package version from upstream I need to create separate branch for that. What upstream release tag are you interested in?

**Review by ruslo — CHANGES_REQUESTED** [review]

File: bin/alex_model.inl (hunk 1 above)
Comment (tonyrewin) [reply bin/alex_model.inl]: i've used https://github.com/liuliu/ccv/tree/stable

Comment (ruslo) [reply bin/alex_model.inl]: > What upstream release tag are you interested in?

Okay, it seems they don't have release tags unfortunately. I have created branch `hunter-0.7-07fc691` based on 07fc691 commit, please rebase this pull request accordingly.

**Comment by tonyrewin** [comment]
i've compared - it is equal, never did rebase cross different repositories, what do you mean?

**Review by ruslo — CHANGES_REQUESTED** [review]
Please rebase your changes against `hunter-0.7-07fc691` branch.

**Comment by ruslo** [comment]
https://github.com/ruslo/hunter/blob/v0.23.214/README.rst

**Pull request closed without merging**